core/src/main/java/io/questdb/client/Sender.java

```diff
@@ -80,6 +88,16 @@ static LineSenderBuilder builder() {
         return new LineSenderBuilder();
     }
 
+    /**
+     * Construct a new Sender instance with default settings.
+     *
+     * @param url full URL to a QuestDB server.
+     * @return a new Sender instance with default settings.
```

Comment:
nit: might be a nice idea to link `url` method somewhere here since it has URL examples.

Reply:
resolving as URL support was removed completely